src/mesh.rs

```diff
@@ -0,0 +1,92 @@
+use geometry::Geometry;
+use hub::Operation;
+use material::Material;
+use object::Object;
+use render::DynamicData;
+
+
+/// [`Geometry`](struct.Geometry.html) with some [`Material`](struct.Material.html).
```

Comment:
just wondering, is there a multi-line doc comment format?

Reply:
No, as far as I know

Reply:
What do you mean by multi-line?

Reply:
By the way, you can put the links at the end if preferable. I think this is recommended practice.

```rust
/// [`Geometry`] with some [`Material`].
///
/// [`Geometry`]: struct.Geometry.html
/// [`Material`]: struct.Material.html
struct Mesh;
```